frontend/src/features/Reportings/Filters/Table/index.tsx

```diff
@@ -56,7 +56,7 @@ export function TableReportingsFiltersWithRef(
 
   const sourceCustomSearch = useMemo(
     () =>
-      new CustomSearch(sourceOptions as Array<Option>, ['label'], {
+      new CustomSearch(sourceOptions as Array<Option<SourceFilterProps>>, ['label'], {
```

Comment:
```suggestion
      new CustomSearch<Option<SourceFilterProps>>(sourceOptions, ['label'], {
```

?

Mais pareil que plus bas : ça serait mieux de typer `sourceOptions` directement ce qui évite ça.